awesomplete.js

```diff
@@ -109,6 +109,14 @@ var _ = self.Awesomplete = function (input, o) {
 		inside: this.container
 	});
 	
+	this.status = $.create("span", {
+		hiddena11y: "",
```

Comment:
Oh, I see how you’re using it. Wouldn’t a class be more appropriate then? Also, can’t we add these properties to something else? Does it have to be an empty span? E.g. perhaps we could apply these properties to the selected item each time and remove them? Would that work?

PS: If accessibility is this verbose and hard, no wonder why most developers ignore it. I care, and I want to make it accessible, but I doubt the average developer would go through this, and it’s sad. Either the specs or the state of implementations have a serious problem.